examples/features/torch_batch_process/batch_inference/compare_with_core_api/core_api_inference.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import print_function
+
+import json
+import logging
+import math
+import os
+import pathlib
+from typing import Any, Optional
+
+import filelock
+import model
```

Comment:
put this after the `import determined as det` clause.  Right now it's hiding among in the "third-party installed packages" section.

Reply:
Umm looks like isort is putting it here, let me see how to bypass this.

Reply:
I checked, and it looks like our other examples have this same shortcoming.

So it's gross, but I don't think you need to fix it today in this pr.